### ISSUE TITLE
Some problem about the experiment results

### ISSUE THREAD

**Issue by RandolphCarter0** [chichidd/BackdoorDynamicDNN#1]
Hello! The method in the paper is excellent and interesting! I try  to reproduce the results of the paper according to the code, but I find that on the cifar10, I run the file `train_cnn.py` `train_sdn.py` `backdoor_sdn.py` with the following command:

`python train_cnn.py --dataset cifar10 --network vgg16 --device cuda:0`
`python train sdn.py --dataset cifar10 --device cuda:0 --network vgg16 --pretrained model_cnn_vgg16.pt`
`python backdoor sdn.py --dataset cifar10 --network vgg16 --device cuda:0 --pretrained model_sdn_vgg16.pt`

I get the acc(top1/5) **0.86/0.99** on the cifar10 test dataset for the vgg16 model which is lower than the accuracy in the paper (**90.8%** in table III) and get the **asr**  `[78.16, 94.40, 98.71, 98.97, 99.60, 99.38, 98.10, 98.47, 99.67, 99.02, 98.19, 98.78, 0.90, 0.98, 0.77]` and the **acc**  `[36.53, 57.83, 65.36, 71.46, 76.35, 80.36, 80.42, 81.37, 81.08, 81.35, 81.11, 80.90, 80.94, 80.86, 80.19]` in the cifar10 test dataset for backdoor sdn vgg16 model, which is different from the results in paper (table II). Did I do something wrong during the operation?
